### PR TITLE
Code Style fixes for administrator/components/com_config/

### PR DESCRIPTION
**Pull request by photodude**
Pull Request for code style issues .
#### Summary of Changes
- Line exceeds 150 characters
- No blank line found after control structure
- Expected "boolean" but found "bool" for function return type
- Whitespace found at end of line
- Blank line found at end of control structure

[Automatically fixed with PHPCS2 fixers](https://github.com/joomla/coding-standards/pull/109)
#### Testing Instructions

Merge by code review
#### Documentation Changes Required

none

### CODE REVIEW

File: administrator/components/com_config/model/application.php [hunk 1]
```diff
@@ -662,7 +661,9 @@ public function storePermissions($permission = null)
 
 		// Get the group, group parent id, and group global config recursive calculated permission for the chosen action.
 		$inheritedGroupRule            = JAccess::checkGroup($permission['rule'], $permission['action'], $assetId);
-		$inheritedGroupParentAssetRule = !empty($parentAssetId) ? JAccess::checkGroup($permission['rule'], $permission['action'], $parentAssetId) : null;
+		$inheritedGroupParentAssetRule = !empty($parentAssetId)
```
Comment (wilsonge) [inline administrator/components/com_config/model/application.php]: This is definitely not right

Comment (photodude) [inline administrator/components/com_config/model/application.php]: Maybe I'm missing something, but what's not right about a multi-line ternary  operator?

Comment (wilsonge) [inline administrator/components/com_config/model/application.php]: I dunno. Maybe definitely was strong but multi-line ternary's feel so wrong to me :/

Comment (photodude) [inline administrator/components/com_config/model/application.php]: It's valid PHP, although we don't have a definitive code style statement on it. 

https://github.com/joomla/coding-standards/issues/119

Comment (photodude) [inline administrator/components/com_config/model/application.php]: @wilsonge Where do we go from here for getting this merged?

Comment (mbabker) [inline administrator/components/com_config/model/application.php]: Personally I'm not a big fan of multi line ternary statements, to me it's a short handed if/else.  If it's starting to expand out, go to a full if/else.

Comment (wilsonge) [inline administrator/components/com_config/model/application.php]: Can we make this either an if/else or go for an inline ternary please. Then happy to merge

Comment (photodude) [inline administrator/components/com_config/model/application.php]: We will have to go for an if/else as I pushed it to multi-line to deal with it exceeding 150 characters.

Comment (wilsonge) [inline administrator/components/com_config/model/application.php]: Fine by me

Comment (photodude) [inline administrator/components/com_config/model/application.php]: @mbabker can we get definitive code style statement on saying multi-line ternary is not allowed. 

https://github.com/joomla/coding-standards/issues/119

Comment (mbabker) [inline administrator/components/com_config/model/application.php]: Opinion logged.

**Comment by photodude** [comment]
@wilsonge This should be ready for consideration now.

**Comment by wilsonge** [comment]
Seems good to me :) Thanks!

**Comment by photodude** [comment]
Thanks @wilsonge

**Pull request merged**